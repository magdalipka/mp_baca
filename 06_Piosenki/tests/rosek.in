3
4
FleetwoodMacGypsy FleetwoodMacDreams FlorenceSweetNothing FlorenceNoLight
5
Piosenka11 piosenka12 piosenka13 piosenka21 piosenka22
15
p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12 p13 p14 p15
