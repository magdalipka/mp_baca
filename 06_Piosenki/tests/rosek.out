FleetwoodMacGypsy FlorenceSweetNothing FleetwoodMacDreams FlorenceNoLight
Fl
Piosenka11 piosenka21 piosenka12 piosenka22 piosenka13

p1 p9 p2 p10 p3 p11 p4 p12 p5 p13 p6 p14 p7 p15 p8
p
